rust/main/utils/run-locally/src/starknet/types.rs

```diff
@@ -102,6 +103,7 @@ impl AgentConfig {
             domain_id: network.domain,
             metrics_port: network.metrics_port,
             mailbox: network.deployments.mailbox.clone(),
+            max_batch_size: 10,
```

Comment:
is this the batch size you'd recommend in prod? right now we set 32 for every chain by default - you can update this by using the `batchSizeOverrides` (example here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/546c6fd460003ae7f474b3d53eb3130e8a8e1a28/typescript/infra/config/environments/mainnet3/agent.ts#L955-L958) in the testnet4 & mainnet3 agent.ts files for all relayer contexts. We should do this before merging if we're uncomfortable with a 32 batch size